Python/find_factors.py

```diff
@@ -0,0 +1,10 @@
+def find_factors(n):
+    """
+    Finds a list of factors of a number
+    """
+    factList = []
```

Comment:
Minor suggestions:

1. May be you can consider using set {} in place of list[], it will help you avoid having repeated elements in the first place, which anyway you are doing in line#10
2. FactList can be initialized with 1 and n e.g. in case of find_factors(16) factList can be initialized with {1,16} and start the range from 2.

Reply:
Implemented the suggestions